DigitalLearningSolutions.Web/Views/Register/Summary.cshtml

```diff
@@ -0,0 +1,107 @@
+        <dt class="nhsuk-summary-list__key">
+          First name
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.FirstName
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="Index">
+            Change<span class="nhsuk-u-visually-hidden"> first name</span>
+          </a>
+        </dd>
+      </div>
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Last name
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.LastName
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="Index">
+            Change<span class="nhsuk-u-visually-hidden"> last name</span>
+          </a>
+        </dd>
+      </div>
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Email
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.Email
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="Index">
+            Change<span class="nhsuk-u-visually-hidden"> email address</span>
+          </a>
+        </dd>
+      </div>
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Job group
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.JobGroup
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="LearnerInformation">
+            Change<span class="nhsuk-u-visually-hidden"> job group</span>
+          </a>
+        </dd>
+      </div>
+      <div class="nhsuk-summary-list__row">
+        <dt class="nhsuk-summary-list__key">
+          Centre
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @Model.Centre
+        </dd>
+        <dd class="nhsuk-summary-list__actions">
+          <a class="nhsuk-link" asp-action="LearnerInformation">
+            Change<span class="nhsuk-u-visually-hidden"> centre</span>
+          </a>
+        </dd>
+      </div>
+    </dl>
+
+    <form method="post" asp-action="Summary">
+      <div class="nhsuk-form-group @termsFormErrorClass">
+        <fieldset class="nhsuk-fieldset" aria-describedby="terms-error">
+          @if (termsError) {
+            <span class="nhsuk-error-message" id="terms-error">
+              <span class="nhsuk-u-visually-hidden">Error:</span> @ViewData.ModelState["Terms"].Errors[0].ErrorMessage
+            </span>
+          }
+          <div class="nhsuk-checkboxes">
+            <div class="nhsuk-checkboxes__item">
+              <input class="nhsuk-checkboxes__input" id="terms" name="terms" asp-for="Terms"/>
+              <label class="nhsuk-label nhsuk-checkboxes__label" for="terms">
+                I have read, understood and agree to abide by the Digital Learning Solutions Terms and Conditions.

```

Comment:
We should make the "Terms and Conditions" a link that opens in a new tab, like on the current website. Note that the NHS guideline is to have "(opens in new window)" after a link that opens in new tab, but HEE have explicitly made a decision not to add this additional text!